src/frontend/src/flows/recovery/displaySeedPhrase.ts

```diff
@@ -43,10 +42,31 @@ const init = (): Promise<void> =>
     displaySeedPhraseContinue.onclick = () => resolve();
 
     const seedCopy = document.getElementById("seedCopy") as HTMLButtonElement;
-    new ClipboardJS(seedCopy).on("success", () => {
-      const seedCopy = document.getElementById("seedCopy") as HTMLButtonElement;
-      displaySeedPhraseContinue.classList.toggle("hidden", false);
-      render(checkmarkIcon, seedCopy);
-      seedCopy.title = "copied";
+    const seedPhrase = document.getElementById("seedPhrase")
+      ?.innerText as string;
+
+    const selectText = (element: HTMLElement) => {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Why disable the eslint check?

Reply:
This is suppressing the not-null assertion. Essentially it is the same as writing `as HTMLElement` (like we do everywhere else). It is just more explicit about suppressing a check.